sdk/core/src/lro.rs

```diff
@@ -49,7 +49,7 @@ pub mod body_content {
         match provisioning_state {
             "Succeeded" => Some(LroStatus::Succeeded),
             "Failed" => Some(LroStatus::Failed),
-            "Canceled" => Some(LroStatus::Cancelled),
+            "Canceled" => Some(LroStatus::Canceled),
```

Comment:
`Canceled` is the spelling defined in the RPC. When I did analysis, I found 8 control plane services that were using `Cancelled`. https://github.com/Azure/azure-resource-manager-rpc/issues/144 .

Reply:
Added support for `Cancelled` and `Canceled` per your research.